include/dali/core/mm/binning_resource.h

```diff
@@ -161,7 +161,7 @@ class binning_resource : public binning_resource_base<Kind, nstatic_bins, Interf
   binning_resource(const SplitPointCollection &split_points,
                    const ResourceCollection &resources,
                    Extra &&extra)
-  : base(split_points, resources), extra_args_(std::forward<ExtraArgs>(extra)) {
+  : base(split_points, resources), extra_args_(std::forward<Extra>(extra)) {
```

Comment:
Weren't there more of those?

Reply:
There other instances of use-after-move warnings turned out to be in the tests where we specifically test moving objects